Python/Query_Script/Readme.MD

```diff
@@ -0,0 +1,16 @@
+### Python Script to get the links of a search query Google Search rank wise
+
+### Modules used
+
+#### Google
```

Comment:
We don't need so many Headings. Follow an elegant structure by taking a look at other Python Scripts.

Reply:
I have followed the readme style of top scripts in python directory, pls tell if any more changes are required